types/setting.go

```diff
@@ -344,6 +348,28 @@ var (
 		ReadOnly:    false,
 		Default:     "10",
 	}
+	SettingDefinitionVolumeAttachmentDeletion = SettingDefinition{
+		DisplayName: "Volume Attachment Deletion Behavior",
+		Description: "Controls when Longhorn should delete a volume attachment on a failed node",
```

Comment:
`Defines the Longhorn action when a Volume stuck with a Deployment Pod on a failed node`